libs/filagui/src/materials/uiBlitExternal.mat

```diff
@@ -0,0 +1,29 @@
+material {
+    name : uiBlitExternal,
+    parameters : [
+        {
+            type : samplerExternal,
+            name : albedo
```

Comment:
What you could have done is using the same material + a bool spec constant to chose which sampler is used at the call site. on the c++ side you'd still need to create 2 materials, but from the same package. It might be easier to maintain?

Reply:
I agree. I pushed a new commit to do this.